geode-core/src/main/java/org/apache/geode/internal/cache/map/AbstractRegionMapPut.java

```diff
@@ -0,0 +1,324 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.geode.internal.cache.map;
+
+import org.apache.geode.cache.DiskAccessException;
+import org.apache.geode.cache.query.internal.index.IndexManager;
+import org.apache.geode.internal.Assert;
+import org.apache.geode.internal.cache.EntryEventImpl;
+import org.apache.geode.internal.cache.InternalRegion;
+import org.apache.geode.internal.cache.RegionEntry;
+import org.apache.geode.internal.cache.Token;
+
+public abstract class AbstractRegionMapPut {
+  private final InternalRegion owner;
+  private final FocusedRegionMap focusedRegionMap;
+  private final EntryEventImpl event;
+  private final boolean ownerInitialized;
+
+  private boolean clearOccurred;
+  private long lastModifiedTime;
+  private RegionEntry regionEntry;
+  private boolean create;
+  private boolean completed;
+
+  public AbstractRegionMapPut(FocusedRegionMap focusedRegionMap, InternalRegion owner,
+      EntryEventImpl event) {
+    if (owner == null) {
+      Assert.assertTrue(false, "The owner for RegionMap " + this + " is null for event " + event);
+    }
+    this.focusedRegionMap = focusedRegionMap;
+    this.owner = owner;
+    this.event = event;
+    this.ownerInitialized = owner.isInitialized();
+  }
+
+  protected boolean isOwnerUninitialized() {
+    return !ownerInitialized;
```

Comment:
Consider eliminating this method. One way to do it is to implement it as:

    return !isOwnerInitialized();

Then inline all uses of `isOwnerUninitialized()`